backend/controllers/taskController.js

```diff
@@ -61,19 +46,14 @@ const getTasksById = asyncHandler(async (req, res) => {
     .catch((err) => {
       if (err) {
```

Comment:
Similar the comment above regarding the redundant `if err` check, please remove it. And remove it from other subsequent files that also use do the `if err` check in the `catch` clause

Reply:
Fixed: Changes are reflected in recent commits